src/components/Router.js

```diff
@@ -75,7 +76,7 @@ export default class Router extends Component {
       ({ pathname, search }) => this.setState({ pathname, search })
     )
 
-    if (isFirecloud() && new URLSearchParams(window.location.search).has('fcredir')) {
+    if (isFirecloud() && _.has('fcredir', qs.parse(Nav.history.location.search, { ignoreQueryPrefix: true, plainObjects: true }))) {
```

Comment:
Can we do this:
```suggestion
    if (_.has('fcredir', qs.parse(Nav.history.location.search, { ignoreQueryPrefix: true, plainObjects: true }))) {
```
so it'll work right on non-prod?